podcastfy/text_to_speech.py

```diff
@@ -38,8 +40,10 @@ def __init__(self, model: str = 'openai', api_key: Optional[str] = None):
 		elif self.model == 'openai':
 			self.api_key = api_key or self.config.OPENAI_API_KEY
 			openai.api_key = self.api_key
+		elif self.model == 'edge-tts':
+			self.edge_tts = edge_tts
```

Comment:
Why do you need to safe the package in a self attribute, can't you simply use edge_tts, which has been imported at the top already, directly in the private method  __convert_to_speech_edge_tts?